### ISSUE TITLE
Banner - Website is optimized for desktop

### ISSUE THREAD

**Issue by c0nb4** [TUB-DVG/wissen-digital-ewb#7]

**Comment by c0nb4** [comment]
Maybe check on which resolution the website still works, so this issue can be linked to: https://github.com/TUB-DVG/wissen-digital-ewb/issues/167

**Comment by nlongn22** [comment]
I've clicked through all pages and found that almost all of them work fine on mobile. There are some pages that are broken on mobile such as these:

1. Environmental page columns
![Screenshot from 2024-10-28 15-02-44](https://github.com/user-attachments/assets/9b08e112-000b-459e-ad1f-873a044cdea3)

2. Criteria catalog
![Screenshot from 2024-10-28 15-02-32](https://github.com/user-attachments/assets/f4f6a4ec-929e-4dd8-baa1-4251de6f941a)
![Screenshot from 2024-10-28 15-02-25](https://github.com/user-attachments/assets/4b714ad5-7b02-437d-95e7-867e7973b1c5)

3. Business model details
![Screenshot from 2024-10-28 15-01-54](https://github.com/user-attachments/assets/e002837e-4388-4ba0-8d1d-b207635812f4)
![Screenshot from 2024-10-28 15-01-32](https://github.com/user-attachments/assets/68042892-6bc9-49a8-b20f-4e99a40e1ac4)


Note that a lot of pages uses the layout like the screenshot above of business models - a simple fix would be just to wrap it on mobile.

In conclusion, I don't think we even need to show the banner as a few CSS fixes should be enough.

**Comment by c0nb4** [comment]
Can you suggest a fix @nlongn22 and present your idea in the next team meeting?

**Comment by c0nb4** [comment]
@nlongn22 can you link the pull request when you are ready? Thank you!
